am-lib/src/main/java/uk/gov/hmcts/reform/amlib/internal/models/ExplicitAccessRecord.java

```diff
@@ -24,6 +26,8 @@
     private final JsonPointer attribute;
     private final Set<Permission> permissions;
     private final String relationship;
+    @Nested
+    private final AccessManagementAudit accessManagementAudit;
```

Comment:
Do we really need to put the auditing information into its own AccessManagementAudit object? Having two fields, lastUpdate and callingServiceName, would make things much easier - no need for @Nested annotations or custom constructors, and would make the db queries less complex

Reply:
i think it should reused across objects. it is good to use object.If we want to extend it it would be easy. else adding every property in every object would be redundant.